docs/content/en/docs/Advanced/build.md

```diff
@@ -47,6 +47,8 @@ helm install kairos-crd kairos/kairos-crds
 helm install kairos-osbuilder kairos/osbuilder
 ```
 
+Among the things deployed by the helm chart, is also an nginx server which is used to
```

Comment:
shall we open up follow-ups e.g. to customize this and mention it here?

Reply:
Sure, if we are sure what kind of customization we want to support, let's open an issue.